docs/reference-architectures/data/enterprise-bi-sqldw.md

```diff
@@ -0,0 +1,402 @@
+
+You can speed up the network transfer by saving the exported data in Gzip compressed format. However, loading compressed files into the warehouse is slower than loading uncompressed files, so there is a tradeoff between faster network transfer versus faster loading. If you decide to use Gzip compression, don't create a single Gzip file. Instead, split the data into multiple compressed files.
+
+### Copy flat files into blob storage
+
+The [AzCopy](/azure/storage/common/storage-use-azcopy) utility is designed for high-performance copying of data to into Azure blob storage.
+
+**Recommendations**
+
+Create the storage account in a region that is closest to where the source data resides. Deploy the storage account and the SQL Data Warehouse instance in the same region. 
+
+Don't run AzCopy on the same machine that runs your production workloads, because the CPU and I/O consumption can interfere with the production workload. 
+
+Test the upload first to see what the upload speed is like. You can use the /NC option in AzCopy to specify the number of concurrent copy operations. Start with the default value, then experiment with this setting to tune the performance. In a low-bandwidth environment, too many concurrent operations can overwhelm the network connection and prevent the operations from fully completing.  
+
+AzCopy moves data to storage over the public Internet. If this isn't fast enough, consider setting up an [ExpressRoute](/azure/expressroute/) circuit. ExpressRoute is a service that routes your data through a dedicated private connection to Azure. Another option, if your network connection is too slow, is to physically ship the data on disk to an Azure datacenter. For more information, see [Transferring data to and from Azure](/azure/architecture/data-guide/scenarios/data-transfer).
+
+During a copy operation, AzCopy creates a temporary journal file, which enables AzCopy to restart the operation if it gets interrupted (for example, due to a network error). Make sure there is enough disk space to store the journal files. You can use the /Z option to specify where the journal files are written.
+
+### Load data into SQL Data Warehouse
+
+Use [PolyBase](/sql/relational-databases/polybase/polybase-guide) to load the files from blob storage into the data warehouse. PolyBase is designed to leverage the MPP (Massively Parallel Processing) architecture of SQL Data Warehouse, which makes it the fastest way to load data into SQL Data Warehouse. 
+
+Loading the data is a two-step process:
+
+1. Create a set of external tables for the data. An external table is a table that points to data stored outside of the warehouse &mdash; in this case, the flat files in blob storage. This step does not move any data into the warehouse.
```

Comment:
With Polybase, I think the "external tables" are really just table definitions / metadata that point to the external source -- correct? (The tutorial on docs.ms.com says "The table definitions are stored in SQL Data Warehouse, but the tables reference data that is stored in Azure blob storage.")
If so, maybe change "is a table that points to data" to ->
"is a table definition that points to data".